stratum/hal/lib/tdi/es2k/es2k_switch_test.cc

```diff
@@ -0,0 +1,363 @@
+// Copyright 2018 Google LLC
```

Comment:
Absent `es2k.BUILD`, I'm going to guess that none of the ported unit tests have been run.

There's more work to be done on the Bazel side.